src/main/java/com/epam/training/array/Array.java

```diff
@@ -5,4 +5,36 @@
   
   private int[] items;
 
+  public Array(int[] items) {
+    this.items = items;
+  }
+
+  public int[] getItems() {
+    return items;
+  }
+
+  public void setItems(int[] items) {
+    this.items = items;
+  }
+  @Override
+  public boolean equals(Object object) {
+    if (this == object) return true;
+    if (object == null || getClass() != object.getClass()) return false;
+    //if (!super.equals(object)) return false;
+    Array array = (Array) object;
+    return java.util.Arrays.equals(items, array.items);
```

Comment:
@Lipskaya Could you add `import` for the util class `Arrays`

Reply:
I fixed it